emails/models.py

```diff
@@ -125,6 +125,7 @@ class Profile(models.Model):
     # TODO: Schema migration to remove null=True
     remove_level_one_email_trackers = models.BooleanField(null=True, default=False)
     onboarding_state = models.PositiveIntegerField(default=0)
+    onboarding_free_state = models.PositiveIntegerField(default=0)
```

Comment:
There are two ways to deal with adding a new field to the model.
- Add `null=True` and `blank=True`
  - Follow-up with the PR to remove these values after applied on production 
- Create a manual SQL query to execute newly added field and its default are properly applied to old entries in the DB

The test was failing because currently this added field will cause an integrity error due to old entries not having `onboarding_free_state` set with `0` as a default ([more details on the error scenario](https://github.com/mozilla/fx-private-relay/blob/main/docs/django-migrations-standards.md#adding-new-model-field)).

Reply:
using commit https://github.com/mozilla/fx-private-relay/commit/b72c2d41a090f255e635c6f42cce1e372a0fb40c and https://github.com/mozilla/fx-private-relay/commit/68604cd3a2699474476d59f9715b8eac7bd3c127 this passes - thanks!